docs/pages/desktop-access/getting-started.mdx

```diff
@@ -315,11 +315,11 @@ windows_desktop_service:
     - "100.104.52.89"
     - "example.com"
   discovery:
-    # This setting will tell Teleport to discover all the hosts connected to the LDAP domain
-    # controller configured above.
+    # The wildcard '*' character tells Teleport to discover all the hosts in
+    # the Active Directory Domain. To refine the search, specify a custom DN.
```

Comment:
We could add an example of a refined search, so that it's clear that "custom DN" means in the form `"DC=example,DC=goteleport,DC=com"`

Reply:
Yeah, this somewhat assumes you know what a DN is. We do give more complete examples in the reference section, so I think it's fine that the "Getting Started" guide uses a reasonable default that's easy to get up and running.